### ISSUE TITLE
syntax guide?

### ISSUE THREAD

**Issue by nathgit** [blissymbolics/blissymbols#8]
How would I get the symbols for I, you, he, she etc; I assumed something like person_(1), person_(2) etc; based on the the sample text & the blissymbols for the words themselves, but that doesn't seem to work. It just shows question marks. Is there an easy to understand guide available for how to figure out what to type to get specific symbols or is one in the works? If not maybe an autocomplete that suggest words as you're typing?  For example if I type "boy", it could suggest possibilities, like "boy,lad", & if you click on one it enters it. This illustrates the main problem I see with Blissymbolics in general. The syntax seems to be very complex or at least unnecessarily unclear, compared to other languages. Even a language like Japanese whose syntax is very strange compared to most languages is consistent & not entirely impossible to understand. Even if you can learn the 2,000 blissymbols, you may not be able to figure out the syntax to correctly write a sentence that anybody could read, let alone read someone else's, defeating the whole purpose of blissymbolics to begin with. Grammar is considered by many linguists to be more time consuming when trying to learn a new language than vocabulary. I don't necessarily agree, but I'm not a linguist either.

**Comment by hlridge** [comment]
Hi nathgit,

I am not the author so, @heatherleaf, fill me in if I missed something....

This script powers a Bliss dictionary (https://spraakbanken.gu.se/karp/#?mode=bliss) through a resource called KARP run by Gothenburg University. While the demo looks like a Bliss editor, it's just a simple demo of the script. The script itself is not an editor (perhaps it should be made more clear). You may build your own editor and use this script to power it.

All Bliss characters (listed in blissdata_chars.js) and Bliss words (listed blissdata_words.js) are looked up by using the exact spelling in those lists. If you make a script, your script may search through all available alternatives based on a search string, like you suggest, and give you matching alternatives. Then you will find "I,me,myself" and "you,yourself" among the hits. This is however not a functionality of this script. 

There are good things with Bliss but I also agree there are problems with Bliss. One among the problems is that there aren't any tools freely available that helps you to easily write Bliss text (considering export options, special cases etc). BlissOnline (https://www.blissonline.se/) has a Bliss Writer tool, but it's behind a paywall. You might find some interesting initiatives here http://blissymbolics.org/index.php/interesting-places, but none is fully functional or updated as far as I can tell.

**Comment by heatherleaf** [comment]
Yes @hl222ih is right : the names "I,me,myself" are really just identifiers and they could have been called 223764 or something like that. Or, as in Unicode where the id of the greek letter σ is U+03C3.

Hopefully this script and database will make it easier for someone who wants to implement an editor, translator or annotator.
